### PR TITLE
Update doc page

### PR DESCRIPTION
**Pull request by jmaitrehenry**
### Description

Update doc style, add space between line, add space inside inline code block, add space between li element.

Update toml documentation page for replacing the docker labels list by a tables.

### CODE REVIEW

**Comment by tboerger** [comment]
Some Screenshot to see the difference should be cool :)

File: docs/css/traefik.css [hunk 1]
```diff
@@ -1,3 +1,22 @@
+body {
+    font-size: 15px;
+    line-height: 24px
```
Comment (ldez) [inline docs/css/traefik.css]: I don't really like size with `px`, I prefer `em` or `rem`

**Comment by jmaitrehenry** [comment]
Before / After space
screen shot 2017-08-22 at 11 17 46 am
screen shot 2017-08-22 at 11 17 45 am

Before / After table
screen shot 2017-08-22 at 11 17 26 am
screen shot 2017-08-22 at 11 17 20 am

I will make the change for using em/rem and not pixel.
And I see I miss a save before commit and the space was on body and not on .main... I will include the fix with the rem/em change.

What do you think for removing the fixed width? We could have more space for long line if we use all the width available.

**Comment by jmaitrehenry** [comment]
@ldez No problem, I made only one PR because it's all related to readability.

**Comment by jmaitrehenry** [comment]
Done!

**Pull request closed without merging**